js/songs/_song_form.html

```diff
@@ -0,0 +1,16 @@
+<form>
+  <p>title:</p> <input type = "text" name="title" data-ng-model="song.title" />
```

Comment:
Few small point on style... 

1. Best practice is to use a `label` tag for form labels, not  `p`
2. No need for spaces between attributes, the `=` and their value
3. I think readability might be improved if you grouped these labels and inputs differently:

```html
<label>artist:</label> 
<input type="text" name="artist" data-ng-model="song.artist" />

<label>album art photo URL:</label> 
<input type="text" name="album_art" data-ng-model="song.album_art" />

<label>genre:</label> 
<input type="text" name="genre" data-ng-model="song.genre"/>

<label>created at:</label> 
<input type="text" name="created_at" data-ng-model="song.created_at"/>

<label>audio URL source:</label> 
<input type="text" name="audio_url" data-ng-model="song.audio_url"/>
```